runtime/doc/vim_diff.txt

```diff
@@ -265,6 +273,8 @@ Lua interface (|if_lua.txt|):
 on cancel and completion respectively) via dictionary argument (replaces all 
 other arguments if used).
 
+|input()| and |inputdialog()| now support user-defined cmdline highlighting.
```

Comment:
Why is only highlighting mentioned, in all of these docs? This is a handler that is called on each input character, correct? That's useful for much more than highlighting.